### PR TITLE
Add escrow list

### PR DESCRIPTION
**Pull request by jrainville**
Merges into https://github.com/status-im/status-teller-network/pull/26

Adds the list of escrows for the seller and the buyer, but if your the seller, you can release and cancel

https://files.gitter.im/status-im/Status-blackops/3XHY/image.png

### CODE REVIEW

File: app/js/features/escrow/saga.js [hunk 1]
```diff
@@ -24,4 +29,63 @@ export function *onCreateEscrow() {
...
+}
+
+export function *onCancelEscrow() {
+  yield takeEvery(CANCEL_ESCROW, cancelEscrow);
+}
+
+export function *doGetEscrows() {
+  try {
+    const eventsSeller = yield Escrow.getPastEvents('Created', {fromBlock: 1, filter: {seller: web3.eth.defaultAccount}});
+    const eventsBuyer = yield Escrow.getPastEvents('Created', {fromBlock: 1, filter: {buyer: web3.eth.defaultAccount}});
```
Comment (alaibe) [inline app/js/features/escrow/saga.js]: Is it safe to go back to block 1? (same question as other PR)

File: app/js/components/EscrowList.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    return escrowStates.released;
+  }
+  if (escrow.canceled) {
+    return escrowStates.canceled;
+  }
+  if (escrow.expired <= Date.now() / 1000) {
+    return escrowStates.expired;
+  }
+  return escrowStates.waiting;
+}
```
Comment (alaibe) [inline app/js/components/EscrowList.jsx]: If we ever show an escrow that could be usefull to be able to reach those function, we could place them for example in:
features/escrow/helper.js

What do you think?

File: app/js/components/EscrowList.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      return <p className="text-warning">Canceled</p>;
+    case escrowStates.expired:
+      return <p className="text-danger">Expired</p>;
+    case escrowStates.waiting:
+    default:
+      return <p className="text-primary">Waiting</p>;
+  }
+}
+
+const EscrowList = (props) => (<Card className="mt-2">
```
Comment (alaibe) [inline app/js/components/EscrowList.jsx]: Could you add the storybook pls?

File: app/js/components/EscrowList.jsx (hunk 2 above)
Comment (jrainville) [reply app/js/components/EscrowList.jsx]: If I understand correctly, you're asking to move those to another file case other escrow files need them? 
If so, yeah it makes sense

**Comment by jrainville** [comment]
Merging as to avoid further conflicts. Story book will be made in a further PR

**Pull request merged**